packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -84,6 +93,21 @@ describe('HCP home page', () => {
 
     await userEvent.unhover(addPatientHoverZone)
     expect(informationTooltip).not.toBeVisible()
+
+    const removeButton = screen.getByRole('button', { name: `Remove patient ${monitoredPatient.profile.email}` })
+    await userEvent.click(removeButton)
+    const removeDialog = screen.getByRole('dialog')
+
+    const dialogTitle = within(removeDialog).getByText(`Remove ${monitoredPatient.profile.firstName} ${monitoredPatient.profile.lastName} from My private practice`)
+    expect(dialogTitle).toBeVisible()
+    const dialogQuestion = within(removeDialog).getByTestId('modal-remove-patient-question')
+    expect(dialogQuestion).toHaveTextContent(`Are you sure you want to remove ${monitoredPatient.profile.firstName} ${monitoredPatient.profile.lastName} from My private practice?`)
+    const dialogInfo = within(removeDialog).getByText('You will no longer have access to their data.')
+    expect(dialogInfo).toBeVisible()
+    const confirmRemoveButton = within(removeDialog).getByRole('button', { name: 'Remove patient' })
+    expect(confirmRemoveButton).toBeVisible()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Shouldn't you click on the confirm button and check that the HTTP call to remove a patient has been made?